cmd/autofix_test.go

```diff
@@ -31,3 +33,28 @@ func TestFixV1Beta1(t *testing.T) {
 	assert.Nil(err)
 	assert.Nil(deep.Equal(correctlyFixedResources, fixedResources))
 }
+
+func TestPreserveComments(t *testing.T) {
+	origFile := "../fixtures/autofix_v1.yml"
```

Comment:
Originally I didn't want to do a text comparison because I thought that would be too specific and assumed that was also the reason the existing tests compared resources instead of file contents, but considering a bug got through, maybe text comparisons would be better for all the tests? I will change this one to call autofix and compare the file contents (it does pass).